src/vs/code/electron-main/windows.ts

```diff
@@ -1023,6 +1026,21 @@ export class WindowsManager implements IWindowsMainService {
 		return res && res[0];
 	}
 
+	private findBestWindow(context: OpenContext, filePaths: IPath[]): VSCodeWindow {
+		const findContainer = context === OpenContext.DESKTOP || context === OpenContext.CLI;
+		return findContainer && this.findContainingWindow(filePaths) || this.getLastActiveWindow();
```

Comment:
I find it hard to understand the precedence logic here without paranthesis, suggest to rewrite to:

`return (findContainer && this.findContainingWindow(filePaths)) || this.getLastActiveWindow();`